app/views/course/assessment/submission/submissions/_statistics.html.slim

```diff
@@ -1,4 +1,6 @@
 - display_grading_results = @submission.published? || can?(:grade, @submission)
+- show_graders = \
+    can?(:grade, @submission) && @submission.answers.reject { |a| a.grader.nil? }.count > 0
```

Comment:
Should we use `@submission.latest_answers`? I think all past answers in missions have a grader of `System`.

Also it's faster to use `any` with a block:  `@submission.answers.any? { |a| a.grader_id.present? }`